tests/unit/records/test_records_jsonschemas.py

```diff
@@ -39,5 +39,6 @@ def test_schemas_are_valid():
     for schemas_dir, _, schemas in os.walk(root_dir):
         schemas_path = os.path.sep.join(schemas_dir.split(os.path.sep)[1:])
         for schema in schemas:
-            schema_path = os.path.join(schemas_path, schema)
-            Draft4Validator.check_schema(fetch_schema(schema_path))
+            if schema.endswith(".json"):
+                schema_path = os.path.join(schemas_path, schema)
+                Draft4Validator.check_schema(fetch_schema(schema_path))
```

Comment:
This could be simplified and prettified to:
```shell
find inspirehep -name "*.json" | while read -r file; do
  python -c "import json; \
    input = json.load(open('${file}', 'r')); \
    output = json.dumps(input, indent=4, sort_keys=True); \
    open('${file}', 'w').write(output.replace(', \n', ',\n') + '\n')"
done
```
since we don't really have other JSON files outside of these folders that don't have to be touched.